scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/RawChange.java

```diff
@@ -52,20 +54,26 @@ default Long getTTL() {
     default Object getAsObject(String columnName) {
         return getAsObject(getSchema().getColumnDefinition(columnName));
     }
-
+    
```

Comment:
trailing whitespaces

Reply:
Eclipse sucks at this particular thing (it constantly adds trailing spaces). I would almost suggest just living with it, because chasing it is such a royal pain. But I can of course rebase.

Reply:
Please rebase. Something must be wrong with your Eclipse setting. Mine is removing trailing spaces automatically with Save Actions.

Reply:
Where on earth do you have that option? I've been waiting for one for ages...

Reply:
https://source.android.com/devices/tech/test_infra/tradefed/development/eclipse#remove_trailing_whitespaces

Reply:
OMG. It is available in "save actions" - only for Java - now.